spec/models/foodstuff_spec.rb

```diff
@@ -45,9 +45,15 @@
 
   # TODO: 別ブランチでテスト
   describe "cuisine_idカラムのバリデーション" do
-    # context "NG" do
-    #   it "同一のcuisine_id, rawmaterial_idであれば無効であること"
-    # end
+    context "NG" do
+      it "同一のcuisine_id, rawmaterial_idであれば無効であること" do
+        foodstuff1 = create(:foodstuff)
+        foodstuff2 = build(:foodstuff, cuisine_id: foodstuff1.id, rawmaterial_id: foodstuff1.rawmaterial_id)
```

Comment:
下記じゃないですか、cuisine_idにfoodstuffのidをつめてるのは間違いかと。

```suggestion
        foodstuff2 = build(:foodstuff, cuisine_id: foodstuff1.cuisine_id, rawmaterial_id: foodstuff1.rawmaterial_id)
```